vars/deployP2Repo.txt

```diff
@@ -0,0 +1,38 @@
+<p>
```

Comment:
Is this shown somewhere on jenkins?

Reply:
https://jenkins.ivyteam.io/job/core_ci/job/master/pipeline-syntax/globals#deployP2Repo